telemetry/qml/FocusableItem.qml

```diff
@@ -0,0 +1,19 @@
+import QtQuick 2.5
+
+FocusScope {
+    id: root
+
+    activeFocusOnTab: true
+
+    MouseArea {
+        anchors.fill: parent
+
+        hoverEnabled: true
+
+        onContainsMouseChanged: {
+            if (containsMouse) {
+                root.forceActiveFocus()
```

Comment:
Is this change necessary? This doesn't add much in terms of accessibility (as screen reader users won't rely on mouse anyway) and makes it possible to accidentally make an unintended choice in this dialog. For example, if one hovers "Yes, send anonymous telemetry..." button and presses Enter, this would enable telemetry in this case even though this probably isn't something that could be intended by this action.